src/utils/auth_utils.ts

```diff
@@ -102,16 +97,16 @@ const checkDeactivatedAccount = (email: string, accessToken: string, statusCode:
 export const isAccountActive = async (email: string, accessToken: string) => {
     const isValidAccount = true;
     const userState = new UserState();
-    userState.set(isValidAccount, false);
+    userState.setValidAccount();
     const userResponse = await createUserWithApi(accessToken);
     if (!userResponse.error) return isValidAccount;
     if (userResponse.statusCode === HttpStatusCodes.UNAUTHORIZED) {
-        userState.set(false, false);
+        userState.setInvalidAccount();
         askAndTriggerSignUp();
         return false;
     }
     const isDeactivated = checkDeactivatedAccount(email, accessToken, userResponse.statusCode);
-	userState.set(isValidAccount, isDeactivated);
+	userState.setDeactivated(isDeactivated);
```

Comment:
Unrelated to this but can you please fix this typo at L124

https://github.com/codesyncapp/codesync-vs/blob/41f7e44cef327b345ee62a5750e292c43e4058d3/src/utils/auth_utils.ts#L124

Reply:
Good catch @saadberry , fixing this